testsuite/api/graphql/introspect.go

```diff
@@ -0,0 +1,82 @@
+//  Copyright (C) 2021 Storj Labs, Inc.
+//  See LICENSE for copying information.
+
+package introspect
```

Comment:
Note, this doesn't need to be in `testsuite` folder. `testsuite` is more for integration tests with the browser and gateway.

Reply:
See satellite/console/consoleweb/endpoints_test.go

Reply:
Thanks for taking a look at this before it was ready for review. I wasn't really sure what we wanted to do for this particular test but it's really just a unit test; so I agree that it belongs with the other endpoint unit tests. I'll move the introspection test and then resolve this review.